app/routes/_gcn.user.email/email_announcements.server.ts

```diff
@@ -12,18 +12,9 @@ import { dedent } from 'ts-dedent'
 
 import type { User } from '../_gcn._auth/user.server'
 import { moderatorGroup } from '../_gcn.circulars/circulars.server'
+import { announcementAppendedText } from '../_gcn.news.email'
```

Comment:
I find it a bit strange that a library module imports a route module. Would you please instead put this definition inside a new file, `email_announcements.ts`?

Reply:
I don't think so. That's the only part that you need in both client and server bundles, right?

Reply:
Yup, just to keep the text consistent so if it changes we only have to update it in one place. I'll push that out shortly